### PR TITLE
Update ITEM_en.tsv

### PR DESCRIPTION
**Pull request by emailboxu**
~4508. 4510+ was edited by someone else previously, skipping potion section.

~4700.

### CODE REVIEW

File: ITEM_en.tsv [hunk 1]
```diff
@@ -4439,7 +4439,7 @@ ITEM_20150428_004825	Superior Drat Bow
 ITEM_20150428_004826	Superior Aston Bow
 ITEM_20150428_004827	Galatunis Bow
 ITEM_20150428_004828	Excel Bow
-ITEM_20150428_004829	Yorgis Pemidian Turret
+ITEM_20150428_004829	Yorgis Femidian Turret
```
Comment (Sunaries) [inline ITEM_en.tsv]: Femidian -> Fedimian

File: ITEM_en.tsv [hunk 2]
```diff
@@ -4642,62 +4642,62 @@ ITEM_20150428_005028	Increases Movement Speed by +4 for 250 seconds. Right-click
...
 ITEM_20150428_005031	Templeshooter Card
-ITEM_20150428_005032	Wizard Panto Cane Decoration
-ITEM_20150428_005033	If you forgot the direction of the cane which installed, it will be heading to you.
+ITEM_20150428_005032	Panto Magician Cane Decoration
+ITEM_20150428_005033	If you forget which direction you planted your claymore, it will be facing you.
 ITEM_20150428_005034	Wire
-ITEM_20150428_005035	You can't find it even if you look closely, as it is very thin.
-ITEM_20150428_005036	Pieces of the dimension
+ITEM_20150428_005035	Hard to see because it is so thin.
+ITEM_20150428_005036	Dimentional Fragment
```
Comment (Sunaries) [inline ITEM_en.tsv]: Dimentional -> Dimensional

File: ITEM_en.tsv [hunk 3]
```diff
@@ -4642,62 +4642,62 @@ ITEM_20150428_005028	Increases Movement Speed by +4 for 250 seconds. Right-click
...
-ITEM_20150428_005066	Meat Loaf that can be easily obtained from Kepo. However, the person can never eat.
-ITEM_20150428_005067	The third revelation found in the Thorn Forest. It's readable.
-ITEM_20150428_005068	The fourth revelation found in the Royal Tomb. It's readable.
-ITEM_20150428_005069	The sixth revelation found in the Cathedral. It's readable.
-ITEM_20150428_005070	There are many rumors that can hear the voice of the King at the Royal Tomb Chapel.
-ITEM_20150428_005071	Reagents for testing Graschius's soul.
-ITEM_20150428_005072	Reagents for testing
-ITEM_20150428_005073	It is said to distinguish the monster which killed the pilgrims to the curse of anger to spary the monster. However, this is still a test reagent.
-ITEM_20150428_005074	The fifth revelation found in the Cathedral. It's readable.
+ITEM_20150428_005039	Torn diary page of ab inhabitant who lived in Andale Village. Read what's inside.
```
Comment (Sunaries) [inline ITEM_en.tsv]: ab -> an
villager is more appropriately than inhabitant, don't you think?

File: ITEM_en.tsv [hunk 4]
```diff
@@ -4642,62 +4642,62 @@ ITEM_20150428_005028	Increases Movement Speed by +4 for 250 seconds. Right-click
...
+ITEM_20150428_005040	Journal of an unknown adventurer who was exploring the Legwyn family mansion. Readable.
+ITEM_20150428_005041	Reply letters from a man named Beikeol. Readable.
+ITEM_20150428_005042	A statement for someone of the Legwyn family who ordered goods. It's for a medicine that has a dangerous side effect. Readable
+ITEM_20150428_005043	A soiled journal discovered in the Waterway. The diary owner's name is illegible. Readable.
+ITEM_20150428_005044	So old it seems like it will crumble ath the slightest touch. Readable.
```
Comment (Sunaries) [inline ITEM_en.tsv]: ath -> at

File: ITEM_en.tsv [hunk 5]
```diff
@@ -4642,62 +4642,62 @@ ITEM_20150428_005028	Increases Movement Speed by +4 for 250 seconds. Right-click
...
+ITEM_20150428_005045	Note left by a pilgrim a very long time ago. Readable.
```
Comment (Sunaries) [inline ITEM_en.tsv]: Note -> A note

File: ITEM_en.tsv [hunk 6]
```diff
@@ -4642,62 +4642,62 @@ ITEM_20150428_005028	Increases Movement Speed by +4 for 250 seconds. Right-click
...
+ITEM_20150428_005046	It seems like it was left here as a last ditch effort by some pilgrim. Readable.
+ITEM_20150428_005047	A part of a pilgrim's torn diary. Readable.
+ITEM_20150428_005048	A hastily scribble drecord of a pilgrim. Readable.
```
Comment (Sunaries) [inline ITEM_en.tsv]: scribble d -> scribbled

File: ITEM_en.tsv [hunk 7]
```diff
@@ -4642,62 +4642,62 @@ ITEM_20150428_005028	Increases Movement Speed by +4 for 250 seconds. Right-click
...
+ITEM_20150428_005049	Will left by a dying pilgrim. Readable.
```
Comment (Sunaries) [inline ITEM_en.tsv]: Will -> A will

File: ITEM_en.tsv [hunk 8]
```diff
@@ -4642,62 +4642,62 @@ ITEM_20150428_005028	Increases Movement Speed by +4 for 250 seconds. Right-click
...
+ITEM_20150428_005050	A burnt journal left here by a pilgrim. Readable.
+ITEM_20150428_005051	A lost diary of some kind. Readable.
+ITEM_20150428_005052	A journal that seems to have been written by someone who was angry. Seems readable upon closer examination.
+ITEM_20150428_005053	A man's journal found in a place where mankind is not allowed to enter. Read what's inside.
```
Comment (Sunaries) [inline ITEM_en.tsv]: A man's journal -> The journal of a person found in a ... 
The Korean string doesn't explicitly say it is a man, but a human person.

File: ITEM_en.tsv [hunk 9]
```diff
@@ -4642,62 +4642,62 @@ ITEM_20150428_005028	Increases Movement Speed by +4 for 250 seconds. Right-click
...
+ITEM_20150428_005054	An order made at Vakarine's command. Read what's inside.
+ITEM_20150428_005055	An order that was written and attached here a long time ago. Readable.
+ITEM_20150428_005056	A document recorded in the time of the Great King Zachariel. Readable.
+ITEM_20150428_005057	Warning notices from hundreds of years ago. Read what's inside.
+ITEM_20150428_005058	Carelessly crumpled message. It seems barely readable.
+ITEM_20150428_005059	Ceremonial ropes that the Panto Magicians were carrying around
```
Comment (Sunaries) [inline ITEM_en.tsv]: Panto Magician. It's not plural.

File: ITEM_en.tsv [hunk 10]
```diff
@@ -4642,62 +4642,62 @@ ITEM_20150428_005028	Increases Movement Speed by +4 for 250 seconds. Right-click
...
-ITEM_20150511_005083	Journal that Vakis has recorded his entire life in. It seems he has enshrined it. It's readable.
-ITEM_20150511_005084	It stated that the future operations along with map. It's readable.
-ITEM_20150511_005085	Scouting Report of Sunset Flag Forest. Even if it's readable, this scratch paper looks creepy.
-ITEM_20150511_005086	File folder that summarizes the study recorded in the tomb. It's readable.
-ITEM_20150511_005087	A documentary that Jonas has devoted a lifetime to study. It's readable.
+ITEM_20150511_005077	Resonance Crystal
+ITEM_20150511_005078	A mysterious crystal dropped by a Vubbe. It resonates when used near a device in the 2nd Mine Lot 2nd Gallery.
+ITEM_20150511_005079	Vakis's completed research reports. Contains information related to the Royal Tombs. Readable.
+ITEM_20150511_005080	An old scroll hidden by the Pyromancer. Readable.
+ITEM_20150511_005081	Black's report. Contains scouted information of the nearby area. Readable.
```
Comment (Sunaries) [inline ITEM_en.tsv]: Black -> Blag

File: ITEM_en.tsv [hunk 11]
```diff
@@ -4642,62 +4642,62 @@ ITEM_20150428_005028	Increases Movement Speed by +4 for 250 seconds. Right-click
...
+ITEM_20150511_005082	Yoke's report. Contains information on the recently increased activity of monsters. Take it to Lieutenant Volda. Readable.
```
Comment (Sunaries) [inline ITEM_en.tsv]: Yoke -> York

**Comment by imcgames** [comment]
Thank you :+1:

**Pull request merged**